tests/nlu/training_data/formats/test_rasa_yaml.py

```diff
@@ -18,32 +18,22 @@
     - how much carbon will a one way flight from [new york]{{"entity": "city", "role": "from"}} to california produce?
```

Comment:
```suggestion
    - how much carbon will a one way flight from [new york]{"entity": "city", "role": "from"} to california produce?
```

Reply:
Why? It's an `f-string`, I dont' think that would work.

Reply:
Ah sorry, my mistake!

Reply:
no worries 😄